students/cjfortu/lesson09/mailroom_00/donor_models.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+"""
+The Mailroom assignment, donor models.
+
+Donor object creation is under the Donor class, while assembling the donor objects is in the DonorCollection class.
+The donor object collection structure is:
+{'name": object'}
+In words, it is a dict with each donor name in string form as the keys, and the donor object as the value.
+
+The remaining features follow the assignment instructions.
+"""
+
+
+class Donor:
+    """Class that holds and accesses information about a single donor."""
+    new_donations = []
+    
+    def __init__(self, donor_name=None, *donations):
+        """Initiate a donor by name and donations."""
+        self.name = donor_name
+        self.donations = [*donations]
+        self.new_donations = [*donations]
```

Comment:
what is the purpose of the two different donations types?

Reply:
I figured there was a chance the donor could make several donations at once.  I would log the individual donations in the data, but send a single letter addressing all donations.  So **donations** is the record of all donations, while **new_donations** is whatever was just donated, either a single donation or group of donations submitted simultaneously.